style.css

```diff
@@ -726,33 +746,186 @@ ul {
     0 0 42px #3cb9fc;
 }
 
-/* MEDIA QUERIES */
+.modal-container {
+  background-color: rgba(0, 0, 0, 0.7);
+  align-items: center;
+  justify-content: center;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 2000;
+  height: 100%;
+  width: 100%;
+  display: none;
+}
 
-@media all and (min-width: 1000px) {
-  .container {
-    /* background-color: blue; */
+.modal-container.showUp {
+  display: flex;
+  flex-direction: column;
+}
 
-    width: 700px;
-    margin: auto;
-    padding: 0;
+.modal {
+  display: flex;
+  flex-direction: column;
+  max-height: 90%;
+  width: 90%;
+  background-color: white;
+  padding: 10px;
+  gap: 15px;
+  animation: modal 0.5s;
+  position: relative;
+  border-radius: 5px;
+}
```

Comment:
**[REQUIRED CHANGES]**

- Hey, you've done well implementing the required changes, however, your modal on mobile is not showing the `see live`,  and `see source` buttons. Here's what you can do to achieve that:

- remove the `max-height and width` here. Based on how you've styled the modal, I think removing these two is necessary as this will make the modal take the whole device height. Also, because you've added some margins auto to the content after the image and you've aligned the item's center, the need of removing the width is quite necessary also.
- This should fix the issue with the buttons not showing.

Good Luck and Happy Coding!